spec/components/admin/participants/details_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+RSpec.describe Admin::Participants::Details, type: :view_component do
+  component { described_class.new profile: participant_profile }
+
+  context "for ECT profile" do
+    let(:mentor_profile) { create :participant_profile, :mentor }
+    let(:participant_profile) { create :participant_profile, :ect, mentor_profile: mentor_profile }
+
+    it "renders all the required information" do
+      expect(rendered).to have_contents(
+        participant_profile.user.full_name,
+        participant_profile.user.email,
+        t(:ect, scope: "schools.participants.type"),
+        mentor_profile.user.full_name,
+        participant_profile.school.name,
+        participant_profile.cohort.display_name,
+      )
+    end
+  end
+
+  # context "for Mentor profile" do
```

Comment:
I know the PR says tests will come later, but just to ping, this needs to be uncommented probably :)